pkg/utils/config.go

```diff
@@ -46,6 +46,9 @@ type CrawlerConfig struct {
 	PersisterPostgresDbname  string                `split_words:"true" desc:"If persister type is Postgresql, sets the database name"`
 	PersisterPostgresUser    string                `split_words:"true" desc:"If persister type is Postgresql, sets the database user"`
 	PersisterPostgresPw      string                `split_words:"true" desc:"If persister type is Postgresql, sets the database password"`
+
+	GCProjectID     string `envconfig:"civil_gc_project_id" desc:"Sets the Google Cloud Project ID name"`
```

Comment:
I was using `PubSubProjectID` in the processor, but this seems better as we may use other GC services, should update the processor similarly (in another PR of course)